config/initializers/assets.rb

```diff
@@ -8,4 +8,4 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( govuk_admin_template/favicon.png )
```

Comment:
This line will be replaced by this:
https://github.com/alphagov/govuk_admin_template/pull/119